### DISCUSSION TITLE
What is the purpose of the neural image function?

### DISCUSSION THREAD

**Discussion by VisionaryMind** [NVlabs/instant-ngp#521] [Q&A]
NeRF works splendidly, even on very small datasets, but I am trying to figure out how to duplicate your results with neural images. It does not appear to be able to increase resolution on low-res images. The Einstein.exr example, for instance -- zooming in does nothing differently than if the image were loaded in a viewer and zoomed in on. Here is what I see:

![image](https://user-images.githubusercontent.com/47434136/164873721-67702e64-6353-4d6e-80ad-006b5a5aaa9e.png)

I have read over your documentation several times, but cannot unravel the mystery of why neural imaging does not work. Even with the gigapixel Tokyo sample, resolution is progressively lost upon zoom-in.

**Comment by Tom94** [comment]
Hi there, the image approximation task is about _representing_ the image through a MLP—not about hallucinating extra detail. So the best case result is indeed that the neural image matches the target image almost exactly.
